Sources/FocusEntity/FocusEntityComponent.swift

```diff
@@ -51,25 +54,28 @@ public struct FocusEntityComponent: Component {
       return nil
     }
   }
+
+    ///Convenient presets
   public static let classic = FocusEntityComponent(style: .classic(color: #colorLiteral(red: 1, green: 0.8, blue: 0, alpha: 1)))
   public static let plane = FocusEntityComponent(
     style: .colored(
-      onColor: .green,
-      offColor: .orange,
-      nonTrackingColor: Material.Color.red.withAlphaComponent(0.2),
-      mesh: FocusEntityComponent.defaultPlane
+        onColor: MaterialColorParameter.color(.green),
+      offColor: MaterialColorParameter.color(.orange),
+        nonTrackingColor: MaterialColorParameter.color(UIColor.red.withAlphaComponent(0.2)),
+        mesh: MeshResource.generatePlane(width: 0.1, depth: 0.1)
```

Comment:
Could you fix the spacing here? (mix of 2 and 4 spaces, rest of the repo uses 2 spaces)